core/src/main/scala/io/chrisdavenport/rediculous/RedisCommands.scala

```diff
@@ -264,6 +264,9 @@ object RedisCommands {
     case class Latest(stream: String) extends StreamOffset {
       override def offset: String = "$"
     }
+    case class LastConsumed(stream: String) extends StreamOffset {
+      override def offset: String = ">"
+    }
```

Comment:
@ChristopherDavenport you mean this change? You are right, this is not backwards-compatible. See:
- https://github.com/lightbend/mima/issues/200

Reply:
But this is I gather, only if pattern matching on the user side? Which seems a rather odd use in this case.

Reply:
@ChristopherDavenport  Anything changes required on this?

Reply:
I think the best way is to just release a new minor. Its limited changes and as you mentioned no one is probably doing this so the only burden is on intermediate libraries. 

Are there any other values here we should add?

Reply:
Answering my own question this looks good.

Reply:
That should be all, I believe.